src/components/calendar/helpers.ts

```diff
@@ -81,6 +113,33 @@ export function getDateFormValue(value: Date | null) {
   return value ? value.toISOString() : null;
 }
 
+/**
+ * Converts a DateDateRangeValue object to FormData with
+ * start and end Date values as ISO 8601 strings.
+ * The keys are prefixed with the host name if it exists
+ * and suffixed with 'start' or 'end' accordingly.
+ *
+ * If the date values are null or undefined, the form data values
+ * are empty strings ''.
+ */
+export function getDateRangeFormValue(
+  value: DateRangeValue | null,
+  host: IgcFormControl
+): FormValueType {
+  if (!value?.start && !value?.end) return null;
+
+  const start = value?.start?.toISOString();
+  const end = value?.end?.toISOString();
+
+  const fd = new FormData();
+  const prefix = host.name ? `${host.name}-` : '';
```

Comment:
If the date range picker does not have a set **name**, then it should **not** participate in the native form submit. Also, what is the behavior in the case where just one part of the range is set? The component skips setting the other at all?